modules/custom-tuning-specification.adoc

```diff
@@ -290,6 +289,7 @@ spec:
 ----
 endif::rosa-hcp-tuning[]
 ifdef::rosa-hcp-tuning[]
+.Example: Machine pool based matching
```

Comment:
> Why have we suddently started using JSON vs. YAML ? YAML is used by all regular OCP docs.

Hello. While I can't answer that question directly as I was never part of that decision, I can tell you that I have seen JSON being used from the time I started here (12/22) so not sure how new it is/when it was implemented.

Reply:
I suspect this will likely need more work. JSON is not as readable as YAML. The configuration ĺooks very strange in this way.  I'm planning to set-up a ROSA with HCP cluster to test this. Who is normally testing this on ROSA HCP? Perhaps we could arrange to share a test environment.

Reply:
> I suspect this will likely need more work. JSON is not as readable as YAML. The configuration ĺooks very strange in this way. I'm planning to set-up a ROSA with HCP cluster to test this. Who is normally testing this on ROSA HCP? Perhaps we could arrange to share a test environment.

Hello @jmencak, understood. I have opened another ticket to address these additional concerns as based on the recent Slack discussions on this issue (https://issues.redhat.com/browse/OSDOCS-9296). I can reach out to our QE  "go-to" person and ask about sharing an environment. 
Sorry I could not be of more help.

Reply:
It could be the the json format is required. I don't know yet, just starting to get familiar with this.  But just trying to create a simple basic tuning configuration, I'm getting:

```
rosa create tuning-config -c <my cluster> --name my-tuning-config --spec-path ./tuning-config.json
E: Expected a valid TuneD spec file: invalid character '\n' in string literal
```

It would be nice who the QE for this is, so that I could work with them. Thank you.

Reply:
Roger that, @jmencak. I have sent a Slack message to @xueli181114 to see if she can assist with that info. Hopefully we can get an answer soon. Thanks!

Reply:
@radtriste will you look at this?

Reply:
Started a thread on this: https://redhat-internal.slack.com/archives/CHWDU0CP9/p1705484998082819

Reply:
I can no longer reproduce the issue. Not sure what the problem was, but with the config:

```
{
  "profile": [
    {
      "data": "[main]\nsummary=Custom OpenShift profile\ninclude=openshift-node\n[sysctl]\nvm.dirty_ratio=\"55\"",
      "name": "tuned-1-profile"
    }
  ],
  "recommend": [
    {
      "priority": 20,
      "profile": "tuned-1-profile"
    }
  ]
}
```

The `tuning-config` is created without any issues.  Appologies for the noise.  What is unfortunate though, is that the rosa client only supports JSON as the input format for the configuration. [Hopefully that changes in the future](https://redhat-internal.slack.com/archives/CHWDU0CP9/p1705494874344759?thread_ts=1705484998.082819&cid=CHWDU0CP9).

@mletalie , thank you for your help and please feel free to close my JSON comment as resolved.

Reply:
Thanks @jmencak, Appreciate the heads up!